components/Select/Select.tsx

```diff
@@ -0,0 +1,30 @@
+import styles from './Select.module.scss';
+
+type SelectProps = {
+    value: string;
+    options: string[];
+    handleChange: (event: React.FormEvent) => void;
+    className?: string;
+};
+
+export const Select = ({
+    value,
+    options,
+    handleChange,
+    className,
+}: SelectProps) => {
+    const optionList = options.map((option: string, index: number) => (
+        <option key={index}>{option}</option>
+    ));
+    return (
+        <select
+            value={value}
+            onChange={handleChange}
+            className={[styles.button, className].join(' ')}
```

Comment:
**Nickpick**: This might be neater as `${styles.button} $ {className}`, `].join(' ')` is nice to use when you've got multiple classnames to insert.

Reply:
I think I will keep this as `join` - using the template results in an `undefined` class being added when the className isn't passed into this component. Join doesn't result in this issue.